content/events/22-23/bouldering.md

```diff
@@ -12,11 +12,16 @@ locationlink: Campinglaan 2, 9000 Gent
 
 # Zeus gaat boulderen
 
-Details worden later nog aangevuld, aanvangstijd kan wijzigen.
+Om als computernerds aan onze gezondheid te denken, zouden we beter wat sporten.
 
-Je zult je op voorhand [hier][boulderen] moeten schrijven.
+Daarom gaat Zeus samen boulderen. We geven ook wat korting!
+We spreken af om **19u30** aan **klimzaal Bleau**. Dat is aan de watersportbaan/Blaarmeersen.
+
+Voor de mensen die samen naar daar willen **fietsen**, dat kan. We vertrekken strikt om 19u00 aan de voorkant van De Sterre S9.
+
+Inschrijven is verplicht. Gelieve je [hier][boulderen] in te schrijven.
```

Comment:
Misschien dit herschrijven als "Inschrijven is verplicht, dit kun je hier", dan heb je maar 1x inschrijven gebruikt op die lijn